github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -272,6 +272,85 @@ presubmits:
           hostPath:
             path: /dev/vfio/
             type: Directory
+  - name: pull-kubevirt-e2e-kind-1.17-sriov-multi
+    skip_branches:
+    - release-\d+\.\d+
+    annotations:
+      fork-per-release: "true"
+      k8s.v1.cni.cncf.io/networks: multus-cni-ns/sriov-passthrough-cni-multi
+    always_run: false
+    optional: true
+    decorate: true
+    decoration_config:
+      timeout: 4h
+      grace_period: 30m
+    max_concurrency: 10
+    labels:
+      preset-dind-enabled: "true"
+      preset-docker-mirror-proxy: "true"
+      preset-shared-images: "true"
+      #sriov-pod: "true" # we control this lane by extended resources (prow/sriov)
+      rehearsal.allowed: "true"
+    spec:
+      nodeSelector:
+        hardwareSupport: sriov-nic-multi
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            - labelSelector:
+                matchExpressions:
+                - key: sriov-pod
+                  operator: In
+                  values:
+                  - "true"
+              topologyKey: kubernetes.io/hostname
+      containers:
+      - image: kubevirtci/bootstrap:v20201119-a5880e0
+        command:
+        - "/usr/local/bin/runner.sh"
+        - "/bin/bash"
+        - "-c"
+        - >
+            apt update &&
+            apt install gettext-base -y &&
+            wget https://dl.google.com/go/go1.13.8.linux-amd64.tar.gz -nv -S &&
+            tar -xzf go1.13.8.linux-amd64.tar.gz -C /usr/local/ &&
+            export PATH=/usr/local/go/bin:$PATH &&
+            export KUBEVIRT_PROVIDER=kind-k8s-sriov-1.17.0 &&
+            export TARGET=kind-k8s-sriov-1.17.0;
+            trap "echo teardown && make cluster-down" EXIT ERR SIGINT SIGTERM;
+            automation/test.sh;
+        # docker-in-docker needs privileged mode
+        securityContext:
+          privileged: true
+        resources:
+          requests:
+            memory: "29Gi"
+            prow/sriov: "1"
```

Comment:
Out of curiosity, where does this come from?

Reply:
https://github.com/kubevirt/project-infra/pull/784
Evgheni applied it manually to one of the nodes which is borrowed for testing